unicodetools/data/ucd/dev/Blocks.txt

```diff
@@ -264,6 +264,7 @@ FFF0..FFFF; Specials
 11600..1165F; Modi
 11660..1167F; Mongolian Supplement
 11680..116CF; Takri
+116D0..116E3; Myanmar Extended-C
```

Comment:
```suggestion
116D0..116FF; Myanmar Extended-C
```

Blocks always end with ...F.
From the consensus from your PR description: “... in a new Myanmar Extended-C block, with range U+116D0..U+116FF ...”